main.nf

```diff
@@ -366,7 +379,7 @@ process bwa {
 
     input:
     file reads from trimmed_reads
-    file index from bwa_index.first()
+    file index from bwa_index
```

Comment:
Are you sure you don't need the `.first()`? I think this was to prevent the process from consuming the channel so that it works with more than one sample..

Reply:
Ok, reverted that one... I had some weird errors when running the test profile and thought this caused exactly this issue, but maybe that was something different :-/